src/util/images.tsx

```diff
@@ -1,10 +1,14 @@
 import { RemoteImage } from "types/image";
+import { LxdStorageVolume } from "types/storage";
 
 export const isVmOnlyImage = (image: RemoteImage) => {
-  return image.variant?.includes("desktop");
+  return image.variant?.includes("desktop") ?? image.server === "local-iso";
```

Comment:
I think this line should be:
```suggestion
  return image.variant?.includes("desktop") || image.server === "local-iso";
```
Otherwise, the local ISO would be detected as a "container only" image. Which would render a Win11 ISO useless.

![image](https://github.com/canonical/lxd-ui/assets/56583786/77f62f1d-ef05-449d-a8d3-d34e15650fa0)

Reply:
This is actually diriment - one cannot proceed further in any way until it's fixed.

![Screenshot from 2023-08-07 18:39:06](https://github.com/canonical/lxd-ui/assets/56583786/ace03692-cc58-4af0-80b5-060907e8b85f)